hack/kubevirt-nightly-build-images.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -ex
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# Used only with the STDCI
+# Builds operator and registry images.
+
+
+LATEST_KUBEVIRT=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest)
+LATEST_KUBEVIRT_IMAGE=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST_KUBEVIRT}/kubevirt-operator.yaml | grep 'OPERATOR_IMAGE' -A1 | tail -n 1 | sed 's/.*value: //g')
+LATEST_KUBEVIRT_COMMIT=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST_KUBEVIRT}/commit)
+go mod edit -require kubevirt.io/kubevirt@${LATEST_KUBEVIRT_COMMIT}
+go mod vendor
+KUBEVIRT_IMAGE=${LATEST_KUBEVIRT_IMAGE} hack/build-manifests.sh
+
+container_id=$(docker ps | grep kubevirtci | cut -d ' ' -f 1)
+registry_port=$(docker port $container_id | grep 5000 | cut -d ':' -f 2)
```

Comment:
Maybe this will be more robust & clear?
```suggestion
registry_port=$(docker inspect $container_id | jq '.[0].NetworkSettings.Ports."5000/tcp"[0].HostPort')
```

Reply:
I'm not 110% sure that jq is available on the test env